modules/auxiliary/scanner/lotus/lotusdomino_hashes.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
```

Comment:
Does this work with Python 2 and 3? Even if it does, it should be `/usr/bin/env python` with a comment clarifying it works with both.

Reply:
It should work with both but I would need to test it to verify.